ibm/service/power/resource_ibm_pi_network_port_attach.go

```diff
@@ -162,19 +172,19 @@ func resourceIBMPINetworkPortAttachRead(ctx context.Context, d *schema.ResourceD
 	networkname := parts[1]
 	portID := parts[2]
 
-	networkC := st.NewIBMPINetworkClient(ctx, sess, cloudInstanceID)
+	networkC := instance.NewIBMPINetworkClient(ctx, sess, cloudInstanceID)
 	networkdata, err := networkC.GetPort(networkname, portID)
 	if err != nil {
 		return diag.FromErr(err)
 	}
 
-	d.Set(helpers.PINetworkPortIPAddress, networkdata.IPAddress)
-	d.Set(helpers.PINetworkPortDescription, networkdata.Description)
-	d.Set(helpers.PIInstanceId, networkdata.PvmInstance.PvmInstanceID)
-	d.Set("macaddress", networkdata.MacAddress)
-	d.Set("status", networkdata.Status)
-	d.Set("network_port_id", networkdata.PortID)
-	d.Set("public_ip", networkdata.ExternalIP)
+	d.Set(Arg_NetworkPortIPAddress, networkdata.IPAddress)
+	d.Set(Arg_NetworkPortDescription, networkdata.Description)
+	d.Set(Arg_PVMInstanceId, networkdata.PvmInstance.PvmInstanceID)
```

Comment:
Same suggestion as above for `Arg_PVMInstanceId` replacement

Reply:
fixed